docs/tubemq_user_guide.md

```diff
@@ -10,9 +10,8 @@
 ### Build distribution tarball
 Go to the project root, and run
 ```bash
-mvn clean package -Dmaven.test.skip
+mvn clean package -DskipTests
 ```
-Each module of the project can also be built separately.
```

Comment:
can you give more details about what's wrong with this removed line?

Reply:
There are dependency relationship between tubemq sub-modules, such as tubemq-server depnedent on tube-client. You can't build tubemq-server without building tubemq-client first since `tubemq-client-0.2-SNAPSHOT.jar` can't be found from anywhere.

Reply:
Thanks for your suggestion.  `mvn install` will package and deploy jar into local repo.  I think this makes sense.